pom.xml

```
@@ -6,7 +6,7 @@
 
   <groupId>io.annot8</groupId>
   <artifactId>annot8-api</artifactId>
-  <version>1.1</version>
+  <version>1.2.0</version>
```

```suggestion
  <version>1.2</version>
```

Api is only versioned at x.y.

We did this because we didn't think it makes sense to patch it (would be for typo I guess but that level hardly matters) but the annon8-components etc are (somewhat) version matched to the API. (1.2.x).